client/prioritizing.go

```diff
@@ -64,6 +67,11 @@ func (p *prioritizingClient) learnGroup(ctx context.Context) error {
 
 // Watch returns new randomness as it becomes available.
 func (p *prioritizingClient) Watch(ctx context.Context) <-chan Result {
+	// prefer the gossip channel for watches
+	if p.gossip != nil {
```

Comment:
can we call this `preferredWatchClient` or similar (and likewise `cdn` as fallback or something), since this module is fairly agnostic about the source of randomness from the sub clients, and we can focus on function.